packages/react/src/components/SwatchColorPicker/ColorPickerGridCell.types.ts

```diff
@@ -80,7 +80,7 @@ export interface IColorPickerGridCellProps {
   /**
    * Handler for when a color cell is clicked.
    */
-  onClick?: (item: IColorCellProps) => void;
+  onClick?: (event: React.FormEvent<HTMLElement>, item: IColorCellProps) => void;
```

Comment:
Unfortunately to avoid a breaking change, you'll need to make the event a second optional parameter.
```suggestion
  onClick?: (item: IColorCellProps, event?: React.FormEvent<HTMLElement>) => void;
```